src/components/attendee/InputSelect/InputSelect.tsx

```diff
@@ -1,17 +1,18 @@
 import React from "react";
-import { FieldError, RegisterOptions, UseFormRegister } from "react-hook-form";
+import {
+  FieldError,
+  RegisterOptions,
+  UseFormRegister,
+  UseFormSetValue,
+} from "react-hook-form";
 import classNames from "classnames";
 
+import { UserProfileModalFormData } from "types/profileModal";
 import { AnyForm, ContainerClassName } from "types/utility";
 
-import styles from "./InputSelect.module.scss";
+import { ProfileLinkIcon } from "../ProfileLinkIcon/ProfileLinkIcon";
```

Comment:
We can create `index.ts` file to avoid the double name path `/ProfileLinkIcon/ProfileLinkIcon`

Reply:
[04f5003](https://github.com/sparkletown/sparkle/pull/3027/commits/04f5003da32c5d46ad3c388886090803250bdac9)